build_wasm.sh

```diff
@@ -29,6 +29,7 @@ FUNCTIONS+='_dedx_get_csda_range_table,'
 FUNCTIONS+='_dedx_get_simple_stp_for_program,'
 FUNCTIONS+='_dedx_get_stp_table_size,'
 FUNCTIONS+='_dedx_fill_default_energy_stp_table,'
+FUNCTIONS+='__dedx_read_density,'
```

Comment:
Looks OK, but in the next iteration (issue) I'd simply expose new function in `libdedx` like `dedx_get_density`.

When implementing current feature keep in mind this future issue https://github.com/APTG/dedx_web/issues/176

Reply:
I've commented in the #176 . I'd argue it should be done in a separate PR since it would change both the data and plot tabs and create some UI-related difficulties

Reply:
I agree that #176 is a matter for a dedicated PR.